docs/AdminGuide.md

```diff
@@ -108,3 +108,54 @@ Example entry:
     "time": "2021-10-05 14:44:59",
 }
 ```
+
+## Update the expiry date of mismatches
+
+1. SSH into toolforge.
+
+    ```bash
+    ssh <your_username>@login.toolforge.org
+    ```
+
+1. The `become` commands allows you to sudo to another user while retaining your personalized environment.
+
+    ```bash
+    become mismatch-finder
+    ```
+
+1. Get into the database using the toolforge alias for accessing Mariadb 
+    ```bash
+    sql tools
+    ```
+
+1. Use the Wikidata mismatch finder database
+
+    ```bash
+    USE s54796__mismatch_finder; 
+    ```
+
+1. Now inside the database we can perform the update operation. 
+
+    ```bash
+    UPDATE import_meta SET expires='<date_of_expiration>' where id = <id_of_entry_to_update>; 
+    ```
+
+    Example:
+
+    ```bash
+    UPDATE import_meta SET expires='2022-11-01' where id = 6
+    ```
```

Comment:
I know these are run in bash but using `sql` as code block language make them prettier imho.
For example:
```sql
UPDATE import_meta SET expires='<date_of_expiration>' where id = <id_of_entry_to_update>; 
```